docker/clients/fedora20/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:20
+FROM fedora:22
```

Comment:
I had to update the repo location in the Dockerfile and realized they were still set to f20 and f21 which won't work for nightly so I updated them to reflect the repos that will actually exist.